src/Command/Adapter/ProviderInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Human Scoring Software
+ *
+ * @author antoinep@taotesting.com
+ * @license See LICENCE.md
+ */
+namespace App\Command\Adapter;
+
+
+interface ProviderInterface
+{
+    /**
+     * return an array of Scorers data
+     *
+     * @param $filePath
+     * @param $symfonyStyle
+     * @return mixed
```

Comment:
Would be great if you could define the type of the @params and also the @return should be Iterator.